### ISSUE TITLE
Do we need to rebuild after a MySQL update?

### ISSUE THREAD

**Issue by johnwc** [infusion/udf_infusion#29]
If we are updating the install to MySQL from say 8.0.26 to 8.0.29, do we need to rebuild and install the library?

**Comment by infusion** [comment]
nope, typically not. The UDF API wasn't changed for a long time so it is fine to drop in the pre-compiled extension also in newer versions.
